src/imageview.cpp

```diff
@@ -98,7 +98,9 @@ void ImageView::onFileDropped(const QString file) {
 }
 
 void ImageView::wheelEvent(QWheelEvent* event) {
-  int delta = event->delta();
+  QPoint angleDelta = event->angleDelta();
+  Qt::Orientation orient = (qAbs(angleDelta.x()) > qAbs(angleDelta.y()) ? Qt::Horizontal : Qt::Vertical);
+  int delta = (orient == Qt::Horizontal ? angleDelta.x() : angleDelta.y());
```

Comment:
I think Qt should restore `QWheelEvent::delta()` or add another method similar to it. Each time we need delta, we should do this calculation to get rid of warnings :(

Reply:
Definitely!
I can try asking in some Qt chats about...